package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.3.58",
+  "version": "2.3.59-rc.3",
```

Comment:
Should this merge? Will it be properly overwritten to `2.3.59`?

Reply:
The `auto-version.yaml` workflow will run a (in this case) `npm version patch` command

Reply:
Just tested locally and if `package.json` has `"version": "2.3.59-rc.3"` then `npm version patch` will indeed change it to `"version": "2.3.59"`

Reply:
I confirmed that it will remove the RC bit.

```
⚡ npm version patch --git-tag-version false
v2.3.59
```